daisi/src/material_flow/model/task.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include "task.h"
 
+using namespace daisi::cpps;
```

Comment:
As mentioned in another comment, we should avoid "using namespace" if possible / it makes sense.